ezbake/config/conf.d/auth.conf

```diff
@@ -45,6 +45,38 @@ authorization: {
             sort-order: 500
             name: "puppetlabs csr"
         },
+        {
+            # Allow the CLI tool to query and update a certificate status,
+            # using the master's hostcert.
+            match-request: {
+                path: "/puppet-ca/v1/certificate_status"
+                type: path
+                method: [get, put, delete]
+            }
+            allow: {
+                extensions: {
+                    pp_cli_auth: true
```

Comment:
Is there a different value than "true" we should be checking against here? I remember we had thought of including the hostname, but apparently the values for extensions in tk-auth have to match the value of the extension exactly, so I wasn't sure how to accomplish that programmatically, considering this is a static file that we ship.